README.md

```diff
@@ -15,7 +15,7 @@
 #### Module add
 
 * Create a new module
-* Format: `module add <module name>`
+* Format: `module add m/<module name>`
```

Comment:
Should be `add module`? For subsequent ones also.

Reply:
Thanks for pointing it out! I have changed add module and add student since they are the ones that have been merged